mkdocs.yaml

```diff
@@ -75,7 +77,8 @@ markdown_extensions:
   - plantuml_markdown:
       server: http://www.plantuml.com/plantuml
       format: svg
-  - pymdownx.arithmatex
+  - pymdownx.arithmatex:
+      generic: true
```

Comment:
Could you tell me the reference documentation for this?

Reply:
And what was the problem? Is this also related to the MathJax issue?

Reply:
Oh I did not pay much attention to it. According to [Arithmatex documentation](https://facelessuser.github.io/pymdown-extensions/extensions/arithmatex/#generic-output-format), this option seems to offer different html rendering approach of mathjax equation. I guess `generic: true` may not be a necessary change this time.

Reply:
So do you want to remove it?

```suggestion
```

Reply:
I tested on my local machine, and found out `genetic: true` is a must option for rendering mathjax. Without this option they were not rendered even if I reloaded the page many times.
I think I can readily ship this PR once I checked https://autowarefoundation.github.io/autoware-documentation/pr-380/ !

Reply:
Understood. Thank you for testing it!